avocado/plugins/xunit.py

```diff
@@ -137,7 +140,8 @@ def render(self, result, job):
             return
 
         max_test_log_size = getattr(job.args, 'xunit_max_test_log_chars', None)
-        content = self._render(result, max_test_log_size)
+        override_job_name = getattr(job.args, 'xunit_job_name', None)
```

Comment:
Since the `default=None` is being defined on argument creation (below), probably this call could be only: `getattr(job.args, 'xunit_job_name')`, couldn't?

Reply:
We use `getattr` because sometimes these plugins are executed with args not really being produced by `avocado run` (mainly in testing, but sometimes even `avocado run --replay` might produce "different" args).

Personally I prefer using `job.args.xunit_job_name` and failures when missing, but I followed the existing style. (note `getattr` without the default fails when arg is missing).